Source/Shaders/BillboardCollectionVS.glsl

```diff
@@ -258,6 +258,24 @@ void main()
     gl_Position = czm_viewportOrthographic * vec4(positionWC.xy, -positionWC.z, 1.0);
     v_textureCoordinates = textureCoordinates;
 
+#ifdef DISABLE_DEPTH_DISTANCE
+    float disableDepthTestDistance = distanceDisplayConditionAndDisableDepth.z;
+    if (disableDepthTestDistance == 0.0 && czm_minimumDisableDepthTestDistance != 0.0) {
+        disableDepthTestDistance = czm_minimumDisableDepthTestDistance;
+    }
+
+    if (disableDepthTestDistance != 0.0) {
+        gl_Position.z = min(gl_Position.z, gl_Position.w);
+
+        bool clipped = gl_Position.z < -gl_Position.w || gl_Position.z > gl_Position.w;
+        float distance = length(positionEC.xyz);
+        if (!clipped && (disableDepthTestDistance < 0.0 || (distance > 0.0 && distance < disableDepthTestDistance)))
```

Comment:
I might have already asked this, but should be store squared distance instead to avoid the `length` above?  Or are you concerned about overflow?  We would treat infinity as a special case.